jina/executors/evaluators/craft/nlp/length.py

```diff
@@ -0,0 +1,26 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from ...decorators import as_aggregator
+from .. import BaseEvaluator
+
+
+class StringLengthEvaluator(BaseEvaluator):
+    """A :class:`StringLengthEvaluator` evaluates the different lengths between doc and groundtruth text
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @property
+    def metric_name(self):
+        return f'Length'
+
+    @as_aggregator
+    def evaluate(self, doc_content: str, groundtruth_content: str, *args, **kwargs) -> float:
```

Comment:
Output can be `int`. Is float necessary?

Reply:
It could be, but since the interface returns `float`